content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -45,6 +45,37 @@ check the documentation for that version.
 
 
 <!-- body -->
+## Install and configure prerequisites:
+
+   (these instructions apply to Linux nodes only)
+
+### Forwarding ipv4 and letting iptables see bridged traffic
+
+Make sure that the `br_netfilter` module is loaded. This can be done by running `lsmod | grep br_netfilter`. To load it explicitly call `sudo modprobe br_netfilter`.
+
+As a requirement for your Linux Node's iptables to correctly see bridged traffic, you should ensure `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config, e.g.
+
+```bash
+   cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
+   overlay
+   br_netfilter
+   EOF
+
+   sudo modprobe overlay
+   sudo modprobe br_netfilter
+
+   # Setup required sysctl params, these persist across reboots.
+   cat <<EOF | sudo tee /etc/sysctl.d/99-kubernetes-cri.conf
+   net.bridge.bridge-nf-call-iptables  = 1
+   net.ipv4.ip_forward                 = 1
+   net.bridge.bridge-nf-call-ip6tables = 1
+   EOF
+
+   # Apply sysctl params without reboot
+   sudo sysctl --system
+```
+
+For more details please see the [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) page.
```

Comment:
Should the link be this instead?

https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements

Reply:
k8s.io links are usually relative paths, as per the guidelines.

Reply:
ack